css/main.css

```diff
@@ -325,3 +325,122 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+/* ### Footer ### */
+footer {
+  display: grid;
+  align-items: center;
+  padding: 80px 0 2px 0;
+}
+
+.footer-head {
+  padding: 0 5% 0 5%;
+}
+
+.footer-head h2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.footer-form {
+  display: flex;
+  flex-direction: column;
+  width: 90%;
+  padding: 0 5% 0 5%;
+}
+
+#form-button {
+  margin-left: 0;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+  background-color: orange;
+  height: 48px;
```

Comment:
## [Optional]

Your button's color is currently a different shade than the one used in Figma. The color they use for button backgrounds is: `#F55800`. 💡 Hint: You can get the specific properties of any element in Figma by clicking on it.